### DISCUSSION TITLE
RuntimeError: Did not find any digitization points of kind FIFFV_POINT_EEG (3) in the info.

### DISCUSSION THREAD

**Discussion by PhD-GOAT** [raphaelvallat/yasa#63] [Q&A] [answered]
https://raphaelvallat.com/yasa/build/html/quickstart.html#eeg-power-in-specific-frequency-bands

I opened the csv file, but it appears like that when I check EEG Fpz-Cz.

**Comment by raphaelvallat** [comment, marked as answer]
This is an issue with data loading in MNE. Could you please copy-paste the exact code that you have used? Make sure to upgrade your version of MNE by typing `pip install -U mne` in a terminal.

**Reply by PhD-GOAT** [reply]
mne was always up to date.

`import mne

raw = mne.io.read_raw_edf('C:/Users/SC4001E0-PSG.edf', preload=True)
print(raw)
print(raw.ch_names)

chan = raw.ch_names
print("print channel :", chan)

print(raw.info['sfreq'])

raw.resample(100)
sf = raw.info['sfreq']
sf

raw.filter(0.3, 45)
data = raw.get_data() * 1e6
print(data.shape)

import pandas as pd

hypno = pd.read_csv("C:/Users/yasa_example_night_young_hypno.csv", squeeze=True)
hypno

import yasa

yasa.plot_hypnogram(hypno)
yasa.sleep_statistics(hypno, sf_hyp=1/30)
counts, probs = yasa.transition_matrix(hypno)
probs.round(3)

import numpy as np

np.diag(probs.loc[2:, 2:]).mean().round(3)

hypno_up = yasa.hypno_upsample_to_data(hypno, sf_hypno=1/30, data=raw)
print(len(hypno_up))

yasa.plot_spectrogram(data[chan.index("EEG Fpz-Cz")], sf, hypno_up)

yasa.bandpower(raw)
yasa.bandpower(raw, relative=False, bands=[(1, 9, "Slow"), (9, 30, "Fast")])
bandpower = yasa.bandpower(raw, hypno=hypno_up, include=(2, 3, 4))
bandpower.to_csv("C:/Users/bandpower.csv")
fig = yasa.topoplot(bandpower.xs(3)['Delta'])`

**Reply by raphaelvallat** [reply]
Where in this code do you get the error? I'm assuming this line:

```python
raw = mne.io.read_raw_edf('C:/Users/SC4001E0-PSG.edf', preload=True)
```

**Reply by PhD-GOAT** [reply]
---> 47 fig = yasa.topoplot(bandpower.xs(3)['Delta'])

read edf is not problem

**Reply by raphaelvallat** [reply]
When using the yasa.topoplot function the index MUST be the channel names (e.g. [‘C4’, ‘F4’] or [‘C4-M1’, ‘C3-M2’]). A list of the accepted montage/channel names can be found here: https://mne.tools/stable/generated/mne.channels.make_standard_montage.html#mne.channels.make_standard_montage

**Reply by PhD-GOAT** [reply]
So, how should I use the physioNet open data?

Then, what if the actual number of channels for clinical data is different?
